plugins/jenkins/test/models/samson/jenkins_test.rb

```diff
@@ -223,4 +247,427 @@ def stub_add_changeset
       jenkins.job_url(96).must_equal "#"
     end
   end
+
+  describe "with auto-config flag" do
+    let(:jenkins_xml_new_job) do
+      <<-XML.strip_heredoc
```

Comment:
yeah, i noticed that. But the xml returned from jenkins client is mostly without indentation and spaces, so just keeping it as returned.